synapse/tests/test_swarm_runtime.py

```diff
@@ -290,3 +290,15 @@ def test_swarm_runtime_has(self):
         self.assertEqual( tufos[0][1].get('foo:bar'), 'baz' )
 
         env.fini()
+
+    def test_swarm_runtime_range(self):
+
+        env = self.getSwarmEnv()
+        answ = env.runt.ask('foo:bar*range=("cat","goat")')
```

Comment:
i'm not entirely sure what this would mean...  "range" applied to strings?  I'd prefer that we limit the range operator to int properties only.  This just feels too counter intuitive and subtle.